### PR TITLE
Add unit tests for application_tests

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: test/unit/webdriver/app_test.py [hunk 1]
```diff
@@ -36,3 +37,98 @@ def test_reset(self):
...
         assert {'sessionId': '1234567890'}, get_httpretty_request_body(httpretty.last_request())
         assert isinstance(result, WebDriver)
+
+    @httpretty.activate
+    def test_install_app(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/install_app'),
+            body='{"value": {"app": ""}}'
```
Comment (KazuCocoa) [inline test/unit/webdriver/app_test.py]: The return value is `'{"value": ""}'`

File: test/unit/webdriver/app_test.py [hunk 2]
```diff
@@ -36,3 +37,98 @@ def test_reset(self):
...
+        assert {'app': 'path/to/app'}, get_httpretty_request_body(httpretty.last_request())
+        assert isinstance(result, WebDriver)
+
+    @httpretty.activate
+    def test_remove_app(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/remove_app'),
+            body='{"value": {"app": ""}}'
```
Comment (KazuCocoa) [inline test/unit/webdriver/app_test.py]: `'{"value": ""}'`

File: test/unit/webdriver/app_test.py [hunk 3]
```diff
@@ -36,3 +37,98 @@ def test_reset(self):
...
+        assert {'app': "com.app.id"}, get_httpretty_request_body(httpretty.last_request())
+        assert result is True
+
+    @httpretty.activate
+    def test_background_app(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/app/background'),
+            body='{"value": {"app": }}'
```
Comment (KazuCocoa) [inline test/unit/webdriver/app_test.py]: maybe here also `'{"value": ""}'`

Comment (manoj9788) [reply test/unit/webdriver/app_test.py]: hmm! okay

**Comment by manoj9788** [comment]
#283

**Pull request merged**